hack/bundle.sh

```diff
@@ -95,7 +95,7 @@ main() {
 	info "Adding additional metadata annotations"
 	cat <<-EOF >>bundle/metadata/annotations.yaml
 		# Annotations for OpenShift version
-		  com.redhat.openshift.versions: "v4.11-v4.14"
+		  com.redhat.openshift.versions: "v4.12-v4.17"
```

Comment:
didn't we decide on 4.14-4.17?

Reply:
for the downstream .. yes. does not apply to the upstream one.